core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -95,7 +95,7 @@ public class RestConfig extends AbstractConfig {
   public static final String SHUTDOWN_GRACEFUL_MS_CONFIG = "shutdown.graceful.ms";
   protected static final String SHUTDOWN_GRACEFUL_MS_DOC =
       "Amount of time to wait after a shutdown request for outstanding requests to complete.";
-  protected static final String SHUTDOWN_GRACEFUL_MS_DEFAULT = "1000";
+  protected static final String SHUTDOWN_GRACEFUL_MS_DEFAULT = "5000";
```

Comment:
we had a lot of tests failing on the `server.stop()` timeout, and bumping this higher seems to allow them to pass. I'm not feeling great about just bumping a timeout w/o understanding what's going on, but there are other logical issues in other tests to look at before we can revisit this.

Reply:
can you add a comment that lower value `1000` led to tests failing ? Probably create a followup if you run out of time to investigate this..

Reply:
sounds good I'll add a comment in a later commit